ControlScheme.json

```diff
@@ -82,5 +82,33 @@
       "Button": 3
     },
     "placeholder"
+  ],
```

Comment:
assume temp -- controlscheme needs to be finalized
see 2024 strategy notes for current buttons